neo4j/README.md

```diff
@@ -2,11 +2,15 @@
 
 ## Overview
 
-This check monitors [neo4j][1] through the Datadog Agent.
-Please verify the metrics and checks that are submitted through this agent. Since Neo4j 4.0 and onward supports multiple databases, some metrics and checks are no longer published.
+Neo4j[1] is the leading enterprise-strength graph database that combines native graph storage, advanced security, scalable speed-optimized architecture, and ACID compliance to ensure predictability and integrity of relationship-based queries. Powered by a native graph database, Neo4j stores and manages data in its more natural, connected state, maintaining data relationships that deliver lightning-fast queries, deeper context for analytics, and a pain-free modifiable data model.
```

Comment:
```suggestion
Neo4j[1] is an enterprise-strength graph database that combines native graph storage, advanced security, scalable speed-optimized architecture, and ACID compliance to ensure predictability and integrity of relationship-based queries. Neo4j stores and manages data in its more natural, connected state, maintaining data relationships that deliver lightning-fast queries, deeper context for analytics, and a pain-free modifiable data model.
```

Slight wording suggestions. Removing 'leading' as we avoid marketing-type language in integrations listings. Removing "Powered by a native graph database" as it's repetitive.